ui/nuxeo-aggregation/nuxeo-checkbox-aggregation.js

```diff
@@ -257,6 +255,19 @@ import { AggregationBehavior } from './nuxeo-aggregation-behavior.js';
       this.setAttribute('tabindex', 0);
     }
 
+    _formatDocCount(docCount) {
+      // Fetch the property value from web-ui-properties.xml
+      const isNumberFormattingEnabled =
+        Nuxeo && Nuxeo.UI && Nuxeo.UI.config && Nuxeo.UI.config.numberFormattingEnabled !== undefined
+          ? Nuxeo.UI.config.numberFormattingEnabled
+          : false; // Default to false if the property is not set
+
+      if (isNumberFormattingEnabled) {
+        return new Intl.NumberFormat().format(docCount); // Apply formatting if enabled
+      }
+      return docCount.toString(); // Return if formatting is disabled
```

Comment:
is toString conversion required ? The previous code doesnt use this conversion

Reply:
In most cases, this avoids unexpected type issues. and won’t impact performance, even if applied repeatedly or on large datasets. It’s a safe addition for consistency without any noticeable drawbacks.

Reply:
if docCount is null, this will throw exception

Reply:
Got the point , removed toString()